builder/builder-next/worker/worker.go

```diff
@@ -50,7 +51,8 @@ import (
 )
 
 func init() {
-	version.Version = "v0.11.0-rc3"
+	version.Version = dockerversion.BuildKitVersion
```

Comment:
Wondering; instead of having this as a package level variable (and adding dockerversion as a dependency here), would it make sense to store this on the Worker itself, and pass it as an argument/option in the Worker constructor?